test/PerfTests/DFPerfTests/host.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.0",
+    "extensions": {
+        "http": {
+          "routePrefix": "tests"
+        },
+        "durableTask": {
+          "hubName": "DFPerfHub",
+          "maxConcurrentActivityFunctions": 400,
+          "maxConcurrentOrchestratorFunctions": 400,
+          "extendedSessionsEnabled": true,
+          "extendedSessionIdleTimeoutInSeconds": 30,
+          "controlQueueBufferThreshold": 1000,
+          "storageProvider": {
+            "partitionCount": 6,
+          }
+        }},
+    "logging": {
+        "applicationInsights": {
+            "samplingSettings": {
+                "isEnabled": true,
+                "excludedTypes": "Request"
```

Comment:
the `logging` section in line 18 seems under indented. Can we format this file? I believe the editor can do this for you as well